src/aca_main.cpp

```diff
@@ -187,14 +205,22 @@ int main(int argc, char *argv[])
   if (g_rpc_protocol == EMPTY_STRING) {
     g_rpc_protocol = UDP_PROTOCOL;
   }
+  if (g_ofctl_command == EMPTY_STRING) {
+    g_ofctl_command = OFCTL_COMMAND;
+  }
+  if (g_ofctl_target == EMPTY_STRING) {
+    g_ofctl_target = OFCTL_TARGET;
+  }
+  
+  // async_grpc_server = new Aca_Async_GRPC_Server();
+  // async_grpc_server_thread =
+  //        new std::thread(std::bind(&Aca_Async_GRPC_Server::Run, async_grpc_server));
 
-  async_grpc_server = new Aca_Async_GRPC_Server();
-  async_grpc_server_thread =
-          new std::thread(std::bind(&Aca_Async_GRPC_Server::Run, async_grpc_server));
-
-  MessageConsumer network_config_consumer(g_broker_list, g_kafka_group_id);
-  rc = network_config_consumer.consumeDispatched(g_kafka_topic);
+  // MessageConsumer network_config_consumer(g_broker_list, g_kafka_group_id);
+  // rc = network_config_consumer.consumeDispatched(g_kafka_topic);
 
+  rc = ACA_OVS_Control::get_instance().control();
```

Comment:
this disable the kafka main thread so that it is running ovs control on the main thread. We may be okay for this PR. The right fix it is keep the kafka main thread and spin up a new thread for ovs control when needed.

Reply:
this command is only for the testing actually. The daemonize function for monitor actually will fork a child thread based on current thread in ovs_control.